app/Enum/CacheKey.php

```diff
@@ -3,6 +3,7 @@
 
 namespace App\Enum;
 
+use App\Models\User;
 use Carbon\Carbon;
 
 class CacheKey
```

Comment:
Ich bin irgendwie gerade nicht so der Freund von der gesamten Klasse, weil ich es ziemlich umständig / unübersichtlich finde, als einfach den Key so an die entsprechende Stelle zu schreiben.

Was sind die Vorteile?

Reply:
Vorteil ist halt, dass man in die `cache` Tabelle gehen kann und mit der IDE genau erkennt, wo die Einträge herkommen. Wenn jeder sein eigenes Schema fährt und an 20 Ecken in der Anwendung auf den Cache zugreift, ist das nicht soo easy herauszufinden.